lit-element/lit-element/shack/ladies_outerwear.json

```diff
@@ -0,0 +1,56 @@
+[
```

Comment:
Can we put the data in a common location?

Reply:
Not too easily, because I need it to be included in the npm packages, but those packages don't share a common directory. Could do something with a script that copies them maybe, but this seems simpler for now. I kind of want to replace the static files with something that generates random ones anyway so that we can play with the scale.